### DISCUSSION TITLE
Who uses PyThaiNLP?

### DISCUSSION THREAD

**Discussion by wannaphong** [PyThaiNLP/pythainlp#514] [General]
Hello everyone! We did like to keep track of who is using PyThaiNLP.
- For company or organization: You can edit [`INTHEWILD.md`](https://github.com/PyThaiNLP/pythainlp/blob/dev/INTHEWILD.md) then send a PR with your company name or @githubhandle or company name with @githubhandle.  If you are not comfortable, you can comment with your company name or @githubhandle in these discussions.
- For users:  you can comment with your name or @githubhandle in these discussions.

Thank you!

**Comment by PanthonImem** [comment]
@PanthonImem
I use PyThaiNLP mainly for tokenization and pre-trained vector embedding. Thank you so much for maintaining this tool!

**Reply by wannaphong** [reply]
Thank you.
